### PR TITLE
Move java gae project

### PR DESCRIPTION
**Pull request by AlexanderLitus**

### CODE REVIEW

**Comment by AlexanderLitus** [comment]
Should I publish core-java and gae-java projects to Maven central repo?

File: core-java-sample/build.gradle [hunk 1]
```diff
@@ -9,9 +9,7 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testCompile project(path: ":core-java", configuration: 'testArtifacts')
 
-    compile project(':core-java')
-    compile project(':java-gae')
-
+    compile project(path: ':core-java');
     compile 'commons-io:commons-io:2.4'
```
Comment (alexander-yevsyukov) [inline core-java-sample/build.gradle]: Why do we need this dependency?

**Comment by alexander-yevsyukov** [comment]
> Should I publish core-java and gae-java projects to Maven central repo? 

I don't think we're ready for pushing the code to the Central repo. We'll do it when we have a framework that can be used for evaluation for real projects.

File: core-java-sample/build.gradle (hunk 1 above)
Comment (AlexanderLitus) [inline core-java-sample/build.gradle]: org.apache.commons.io.FileUtils.deleteDirectory method is used in FileSystemStorageFactory. Unfortunately, there is no such method in java.nio (its delete method can delete only empty dirs).

**Pull request merged**